sources/packages/api/src/services/institution/institution.service.ts

```diff
@@ -209,8 +208,10 @@ export class InstitutionService extends RecordDataModelService<Institution> {
     // Get admin authorization type.
     const authorizationType =
       await this.institutionUserTypeAndRoleRepo.findOneOrFail({
-        type: InstitutionUserType.admin,
-        role: null,
+        where: {
+          type: InstitutionUserType.admin,
+          role: null,
```

Comment:
Same comment from before. Should we use `IsNull()` instead?

Reply:
It did not throw any error or warnings, but thought of taking these changes in the next PR. But will change it as its comments are given now.